.github/workflows/build_test.yml

```diff
@@ -25,8 +25,8 @@ jobs:
       fail-fast: false
       matrix:
         env:
-          - { COMPILER: "gcc",   COMPILER_VERSION: "11", LINKER: "bfd",  CRYPTOLIB: "gcrypt"  }
```

Comment:
Hmm, I think I would prefer to drop something else, e.g. gcc-12 or one of the other clang options. Let's keep gcc-11 working as the baseline version.